### PR TITLE
ES: Fix content check in ImportTitleDone

### PR DESCRIPTION
**Pull request by leoetlino**
ImportTitleDone only checks if all required contents have been imported for system titles.

This fixes the system menu not being able to recreate title directories by using title import functionality when copying a save back to the NAND.

This PR also adds more logging to title import functions to make troubleshooting easier. Review separately.

### CODE REVIEW

File: Source/Core/Core/IOS/ES/TitleManagement.cpp [hunk 1]
```diff
@@ -438,25 +458,10 @@ ReturnCode ES::ImportTitleDone(Context& context)
...
-          return shared_content_map.GetFilenameFromSHA1(content.sha1).has_value();
-
-        // Note: the import hasn't been finalised yet, so the whole title directory
-        // is still in /import, not /title.
-        const std::string path = Common::GetImportTitlePath(title_id) +
-                                 StringFromFormat("/content/%08x.app", content.id);
-        return m_ios.GetFS()->GetMetadata(PID_KERNEL, PID_KERNEL, path).Succeeded();
-      });
-  if (!has_all_required_contents)
+  if (title_id - 0x100000001LL <= 0x100 &&
```
Comment (BhaaLseN) [inline Source/Core/Core/IOS/ES/TitleManagement.cpp]: Don't we have a constant somewhere for 1-1?

Comment (leoetlino) [reply Source/Core/Core/IOS/ES/TitleManagement.cpp]: We have one for 1-2 (the System Menu), but none for 1-1 (that title doesn't seem to exist?)

Comment (leoetlino) [reply Source/Core/Core/IOS/ES/TitleManagement.cpp]: It's what ES checks, not sure why Nintendo bothered checking the TID falls exactly in the range [1-1, 1-101] and didn't just check the higher 32 bits (title type).

Comment (BhaaLseN) [reply Source/Core/Core/IOS/ES/TitleManagement.cpp]: Good enough for me then.

**Pull request merged**